cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
It actually does.. if lock was acquired quickly we never print it. Our real estate on console is minimal we should just avoid printing it if it's not needed. So we needed this style.